src/main/resources/db/migration/V110__AccessionCollectors.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE accession_collectors (
+    accession_id BIGINT NOT NULL REFERENCES accessions ON DELETE CASCADE,
+    position INTEGER NOT NULL,
+    name TEXT NOT NULL,
+    PRIMARY KEY (accession_id, position),
+    CHECK (position >= 0)
+);
+
+-- Add cascading deletes to accession_secondary_collectors so we can get rid of the DELETE statement
+-- on that table in accession deletion; this will make it safer to stop writing to
+-- accession_secondary_collectors in a later code change.
+ALTER TABLE accession_secondary_collectors
+    ADD CONSTRAINT accession_secondary_collectors_accession_id_fkey
```

Comment:
This constraint name is different from the one that's dropped below (`collectors` vs. `collector`).